bluesky_kafka/supervisor/supervisorctl.conf

```diff
@@ -0,0 +1,3 @@
+[supervisorctl]
```

Comment:
Rather than adding a `[program:mongo_consumer]` to `supervisord.conf` I think it is cleaner to add this to `/etc/supervisor/supervisord.conf`
```
[include]
files = /etc/supervisor/conf.d/*.conf
```
and have a standalone file `/etc/supervisor/conf.d/mongo_consumer.conf` with the `[program:mongo_consumer]` section. Someone would have to copy `mongo_consumer.conf` from this repository to that location.

This is how Dan set up supervisor at SST. The big advantage is that when we have more than one program being managed by supervisor we don't have to edit `supervisord.conf`, just copy a new file into the `conf.d` directory.

It looks like this:

```
jlynch@xf07id1-ws19:~$ ls -l /etc/supervisor/
total 12
drwxr-xr-x 2 root root 4096 Jul 10 12:50 conf.d
-rw-r--r-- 1 root root 1188 Jan 13  2020 supervisord.conf
jlynch@xf07id1-ws19:~$ ls -l /etc/supervisor/conf.d/
total 8
-rwxr-xr-x 1 root root 242 Jan 13  2020 bluesky-0MQ-proxy.conf
-rwxr-xr-x 1 root root 287 Jul  9 21:24 suitcase-worker.conf
```

Reply:
But isn't simpler to skip the installation step, and just use -c to point to the conf file in the bluesky-kafka repo? Like I added in the readme. I can rename the supervisord.conf to mongo_consumer.conf. Does that seem okay?

Reply:
I don't understand the benefit of "installing" the supervisord.conf files. Could you explain a little more?
Thanks!

Reply:
For development it may be simpler to specify a supervisor configuration file with -c, but in production I think it makes sense to use system-wide supervisor installation with /etc/supervisor/supervisor.conf and /etc/supervisor/conf.d/*.conf.